### DISCUSSION TITLE
Metronome: Setting or mod to make a metronome sound while playing

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#19857] [Ideas]
A mod that adds a metronome to the beat while you play, some maps have circles mapped to scilence or just maps with terrible music, where it would be nice to just listen to a metronome.
There is already a similar feature in the Muted mod, but that also mutes the song after a while, which is not the intention

**Comment by SevenChords** [comment]
in the mod customisation you can change the metronome mod to not mute hitsounds

**Reply by Doksha** [reply]
The metronome only comes in as the music fades, Im suggesting a mod that specifically adds a metronome at all times (without affecting volume)
Maybe simply separate the metronome function from Muted, with its own customization options
